src/components/Calculator.jsx

```diff
@@ -3,7 +3,9 @@ import React from 'react';
 const Calculator = () => (
   <div>
     <div className="calculator">
-      <div className="screen">0</div>
+      <div className="screen">
+        0
+      </div>
       <div className="calculator-buttons">
         <button type="button">AC</button>
         <button type="button">+/-</button>
```

Comment:
Good job setting up your first component 👍, however there is a tiny improvement to make according to the requirements. The calculator component is expected to have one or more child component in order to get you started with working with multiple components.

HINT: You can create a file for the screen and put the screen related markup in that file and then import it into this file and render it as `<Screen />`